src/core.js

```diff
@@ -1091,42 +1051,15 @@ define([
                     msg = "Parsing Error. Please check that your form is valid XML.";
                 }
 
-                // this button doesn't seem to actually exist
-                // todo: fix
```

Comment:
this is a Mike White leftover, happy to put it back if someone else knows what it's talking about...